README.md

```diff
@@ -22,6 +22,16 @@ $> inertia init
 
 $> inertia remote add glcoud 35.227.171.49 -u root -i /path/to/my/.ssh/id_rsa
 Remote 'glcoud' added.
+$> inertia remote add gcloud
+Enter location of PEM file (leave blank to use '/Users/yourspecialname/.ssh/id_rsa'):
+/path/to/my/.ssh/id_rsa
+Enter IP address of remote:
+35.227.171.49
+Enter user:
+root
```

Comment:
So the user can in principle be parsed from the `/Users/yourspecialname/.ssh/id_rsa.pub` file. And in reality, we need to do _some_ verification on whether the user in the public key matches this name. That helps us avoid this type of failure:

```bash
inertia gcloud init
[Permission denied: public key]
```

I bet one of these go libraries we already have as a dependency lets you look at the public key user for a private key 🤔

One nice way to fix:

```
Enter user (press Enter to use default user for this PEM <default_user_we_parsed>):
```

Reply:
@chadlagore If we can read the user of the key from the key, should we even allow manual input of a user?

Reply:
Its true. If they enter the wrong username, we have no way to recover. Ideally, they don't have too. Another way is they could configure their ssh root once, and we always go rummaging around in there for keys.

Reply:
Might leave this for another ticket

Reply:
👍